main/views.py

```diff
@@ -3,10 +3,11 @@
 """
 import json
 
+from django.contrib.auth.views import redirect_to_login
 from django.shortcuts import render, redirect, reverse
 from django.views.decorators.csrf import csrf_exempt
 from django.views.generic import TemplateView
-
+from django.conf import settings
```

Comment:
Please remember to keep following the sequence of imports in the order. So it should come before the `redirect_to_login` import.